mint-client/src/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "src"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+[[bin]]
+name = "src"
+path = "main.rs"
+
+[lib]
+name = "src"
+path = "lib.rs"
```

Comment:
I didn't notice that too... Must have happend when I struggled initializing (did use cargo init and new but thought I deleted everything)

Reply:
you can just delete this file. It is not being used anywhere.